src/docs/user/references/Variables.adoc

```diff
@@ -180,6 +181,37 @@ For example, a variable `MY_OPTIONAL_INTEGER` defined as the model `PA:INTEGER?`
 
 All the available model syntaxes, except `PA:NOT_EMPTY_STRING` support to be defined as *optional*.
 
+====== Variable Model using Resource Manager data
+
+ProActive Resource Manager provides a set of REST endpoints which allow to create dynamic models based on the Resource Manager state.
+
+These models are summarized in the following table. Models returned are PA:LIST types which allow to select a value in ProActive portals through a drop-down list.
+The list always contain an empty value choice.
+
+.Resource Manager Models
+[%header,cols=4*]
+|===
+|Metric Name
+|Description
+|Model Syntax
+|Example returned data
+
+|Hosts
+|All machine host names or ip addresses registered in the Resource Manager
+|`PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/hosts)`
+|PA:LIST(,try.activeeon.com,10.0.0.19)
+
+|Node Sources
+|All node sources registered in the Resource Manager
+|`PA:MODEL_FROM_URL(${PA_SCHEDULER_REST_PUBLIC_URL}/rm/model/nodesources)`
+|PA:LIST(,Default,LocalNodes,GPU,Kubernetes)
+
+|Tokens
+|All tokens registered in the Resource Manager (across all ProActive Nodes registered). See <<_node_access_token>>.
```

Comment:
Shouldn't be across all registered ProActive Nodes ?

Reply:
seems equivalent to me

Reply:
but ok